### ISSUE TITLE
Lot's of tests being skipped for ReFrame tests of compat layer

### ISSUE THREAD

**Issue by ocaisa** [EESSI/compatibility-layer#207]
I noticed in #206 that a lot of [ReFrame tests are actually being skipped](https://github.com/EESSI/compatibility-layer/actions/runs/9649122336/job/26611871275#step:8:26) for the same reason:
```
WARNING: skipping test 'EchoTest': name error: compat_layer.py:38: name 'eessi_repo_dir' is not defined
```

**Comment by boegel** [comment]
@bedroge fixed with #208?

**Comment by ocaisa** [comment]
Yes, fixed
